### ISSUE TITLE
Missing img/icons.atlas

### ISSUE THREAD

**Issue by Kenoshen** [bitDecayGames/Helm#229]
It might be a problem on my end, but I can't seem to find the img/icons.atlas file.  Did it not get committed?  Or is it generated somehow?

**Comment by MondayHopscotch** [comment]
Yeah, there's a HelmPacker that you should run. You might ahve to set the
working directory to get it to work properly.

On Tue, Oct 17, 2017 at 1:03 AM, Michael Wingfield <notifications@github.com
